migrate/src/test/scala/migrate/MigrateScalacOptionsSuite.scala

```diff
@@ -0,0 +1,47 @@
+package migrate
+
+import org.scalatest.funsuite.AnyFunSuiteLike
+import scalafix.testkit.DiffAssertions
+
+class MigrateScalacOptionsSuite extends AnyFunSuiteLike with DiffAssertions {
+  test("Empty ScalacOptions list") {
+    val input = Nil
+    val res   = ScalaMigrat.migrateScalacOptions(input)
+    assert(res.notParsed.isEmpty)
+    assert(res.specificScala2.isEmpty)
+    assert(res.migrated.isEmpty)
+  }
+  test("ScalacOptions 1") {
+    val input = Seq(
+      "-encoding",
+      "UTF-8",
+      "-explaintypes",
+      "-Yrangepos",
+      "-feature",
+      "-language:higherKinds",
+      "-language:existentials",
+      "-Xlint:_,-type-parameter-shadow",
+      "-Xsource:2.13",
+      "-unchecked",
+      "-deprecation",
+      "-Xfatal-warnings",
+      "-Wunused:patvars"
+    )
+    val res               = ScalaMigrat.migrateScalacOptions(input)
+    val expectedSpecific2 = Seq("-Yrangepos", "-Xlint:_,-type-parameter-shadow", "-Wunused:patvars")
+    val expectedMigrated = Seq(
+      "-encoding UTF-8",
```

Comment:
That's misleading because it would produce a message that cannot be copy pasted:
```
You can update your scalacOptions for Scala 3 with the following settings.
["-encoding UTF-8", "-feature", ....]

Reply:
I agree. I was thinking of improving this in my next commit while adding the plugin part. But I can do that here too.